components/automate-ui/src/app/modules/infra-proxy/tree-table/component/tree-table.component.html

```diff
@@ -0,0 +1,25 @@
+<table mat-table [dataSource]="dataSource" [ngClass]="formatElevation()">
```

Comment:
do we need the `formatElevation()` function?  It kind of looks like its not really doing anything, and when I look at the markup it looks like it just adds a shadow.

Reply:
I have removed this function :)